draft-ietf-quic-recovery.md

```diff
@@ -245,6 +245,15 @@ QUIC specifies a time-based definition to ensure one or more packets are sent
 prior to a dramatic decrease in congestion window; see
 {{persistent-congestion}}.
 
+### The Minimum Congestion Window is 2 Packets
+
+QUIC recommends that the minimum congestion window be 2 packets instead
+of the 1 packet minimum in TCP. A minimum of 2 packets avoids waiting for a
+delayed acknowledgement, which can substantially decrease throughput,
+particularly if the max_ack_delay is larger than the round trip time.
+A 2 packet minimum congestion window also avoids waiting for a probe
+timeout(see {{pto}}) every time a single packet is lost.
```

Comment:
I don't think that this is saying exactly what you want to say.

The point here, if I understand it, is that when an endpoint detects persistent congestion and reduces the window to the minimum, having two packets means that there is a far lower chance of BOTH of those packets being lost.  That greatly improves the likelihood that an acknowledgment will be returned and the connection can start to claw itself out of the hole it found itself in.

The point about sending a probe on PTO is only a consequence of the above; I wouldn't bother mentioning PTO.